images/ubuntu/19.04/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/ubuntu:19.04
+
+ENV NAME=ubuntu-toolbox VERSION=19.04
+LABEL com.github.debarshiray.toolbox="true" \
+      com.redhat.component="$NAME" \
+      name="$FGC/$NAME" \
+      version="$VERSION" \
+      usage="This image is meant to be used with the toolbox command" \
+      summary="Base image for creating Ubuntu 19.04 toolbox containers" \
+      maintainer="Debarshi Ray <rishi@fedoraproject.org>"
+
+COPY README.md /
+
+RUN apt-get update
+
+RUN apt-get -y upgrade
+
+COPY extra-packages /
+RUN apt-get -y install $(cat extra-packages | xargs)
+RUN rm /extra-packages
+
+RUN sed -i -e 's/ ALL$/ NOPASSWD:ALL/' /etc/sudoers
```

Comment:
Is this really necessary? Toolbox itself calls `passwd --delete root` inside the container to delete the `root` password.

Reply:
Unfortunately yes. For some reason I haven't figured out - it's required for sudo to work.

Reply:
I see. The other option is to add the `nullok` option to the `PAM(8)` configuration.